packages/nimble-components/src/patterns/button/tests/doc-strings.ts

```diff
@@ -1,3 +1,28 @@
+export const appearanceDescription = `This attribute affects the appearance of the button.
+
+<details>
+    <summary>Appearance Usage</summary>
+    There is a hierarchy to which button appearance should be used.
+    <ul>
+        <li>For appearance \`ghost\`:
+            <br/>This is the default and standard option for buttons. They create an open / airy feel, and fit comfortably in tight spaces.
+            <br/>Be careful when using that the surrounding context does not confuse this button for emphasized body text, tabs or a standalone links. 
+            <br/>Use this in combination with a primary outline and primary block buttons to create a hierarchy of importance.
+        </li>
+        <br/>
+        <li>For appearance \`outline\`:
+            <br/>This is an alternative standard button when a ghost button is not suitable, and a more visually direct button is needed. 
+            <br/>This can also create a light and airy feel. Use in combination with ghost buttons, not block buttons to create hierarchy.  
```

Comment:
```suggestion
            <br/>This can also create a light and airy feel. Use in combination with ghost buttons to create hierarchy. Do not use in combination with block buttons. 
```